source/common/http/filter/ratelimit.cc

```diff
@@ -117,9 +117,16 @@ void Filter::complete(Envoy::RateLimit::LimitStatus status) {
     break;
   case Envoy::RateLimit::LimitStatus::OverLimit:
     cluster_->statsScope().counter("ratelimit.over_limit").inc();
-    Http::CodeUtility::ResponseStatInfo info{config_->scope(), cluster_->statsScope(), EMPTY_STRING,
-                                             *getTooManyRequestsHeader(), true, EMPTY_STRING,
-                                             EMPTY_STRING, EMPTY_STRING, EMPTY_STRING, false};
+    Http::CodeUtility::ResponseStatInfo info{config_->scope(),
```

Comment:
I looked around and I can't find an obvious way to fix this, unless we change AlignAfterOpenBracket to DontAlign which changes a whole bunch of other things. Will leave for now and we can see if we can change this later. I agree it would be nice if it were like a normal function call.